site/src/components/WorkspaceDeletion/utils.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as TypesGen from "api/typesGenerated"
+import * as Mocks from "testHelpers/entities"
+import { displayImpendingDeletion } from "./utils"
+
+describe("util > workspace deletion", () => {
+  describe("displayImpendingDeletion", () => {
```

Comment:
I think we could let only this describe here. The previous one didn't "make sense" to me since we are only testing this function.